start.js

```diff
@@ -0,0 +1,8 @@
+const app = require('./servers/mongo/server-mongo.js');
+
+const mongoose = require('mongoose')
+mongoose.connect('mongodb://localhost/qa', {useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false, autoIndex: false});
+
+const PORT = 3000;
+
+app.listen(PORT, () => {console.log(`Listening at ${PORT}`)})
```

Comment:
Where does this script get used?

Reply:
that is for my 'production' db and i have another one for my stress tests